lib/jekyll/document.rb

```diff
@@ -1,15 +1,16 @@
 # encoding: UTF-8
 
 module Jekyll
+  # rubocop:disable ClassLength
```

Comment:
@ayastreb increase the max class size to `300` in `.rubocop.yml` that was an arbitrary number I just randomly set and @parkr decided to keep, I see no problem with raising it.